release/alpine-full/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-alpine AS builder
+FROM python:3.9-alpine AS builder
```

Comment:
why changing only alpine and what's the background?

Reply:
CLUS DevNet containers are alpine with python 3.9

Reply:
got it. thanks for the info.